app/services/idv/phone_step.rb

```diff
@@ -146,5 +147,23 @@ def extra_analytics_attributes
         vendor: idv_result.except(:errors, :success),
       }
     end
+
+    def run_job(document_capture_session)
+      callback_url = Rails.application.routes.url_helpers.address_proof_result_url(
+        document_capture_session.result_id,
+      )
+
+      if Figaro.env.proofing_lambda_http_callback == 'true'
+        LambdaJobs::Runner.new(job_name: nil, job_class: AddressProofJob,
+                               args: { applicant_pii: applicant, callback_url: callback_url }).run
+      else
+        LambdaJobs::Runner.new(
+          job_name: nil, job_class: AddressProofJob,
+          args: { applicant_pii: applicant, callback_url: callback_url }
+        ).run do |idv_result|
+          document_capture_session.store_proofing_result(idv_result)
+        end
+      end
+    end
```

Comment:
My vision for this `LambdaRunner` was that it would handle the local-vs-remote dispatch bit, so this would be more like this:

```suggestion
    LambdaJobs::Runner.new(
      job_name: nil, job_class: AddressProofJob,
      args: { applicant_pii: applicant, callback_url: callback_url }
    ).run do |idv_result|
      document_capture_session.store_proofing_result(idv_result)
    end
```